tests/utils/network.js

```diff
@@ -104,6 +105,9 @@ export class LocalNetwork {
                 sfu_channel_uuid: channelUUID,
                 session_id: sessionId,
             }),
+            {
+                channelUUID,
+            }
```

Comment:
Could be a single LOC with `{ channelUUID }`, don't hesitate to help prettier a bit ;)